### PR TITLE
copr: Add vectorize Trim3Args

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu shui.yu@126.com
PCP #6959 

###  What have you changed?
Add vectorize Trim3Args.

###  What is the type of the changes?
- New feature (a change which adds functionality)

###  How is the PR tested?
- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
No.

###  Does this PR affect `tidb-ansible`?
No.

###  Refer to a related PR or issue link (optional)
#6959, #5751

### CODE REVIEW

File: components/tidb_query_common/src/string.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
```
Comment (zhongzc) [inline components/tidb_query_common/src/string.rs:1]: Maybe the module `tidb_query_shared_expr` is an appropriate location to place.

**Comment by zhongzc** [comment]
I suggest picking up challenge from #6959 and #6958 first

File: components/tidb_query_common/src/string.rs (hunk 1 above)
Comment (pingyu) [reply components/tidb_query_common/src/string.rs]: @zhongzc Comment addressed. PTAL~

**Comment by zhongzc** [comment]
PTAL @breeswish

**Pull request merged**